packages/h5wasm/src/h5wasm-api.ts

```diff
@@ -111,6 +111,19 @@ export class H5WasmApi extends ProviderApi {
     file.close();
   }
 
+  public async getSearchablePaths(root: string): Promise<string[]> {
+    const file = await this.file;
+
+    const h5wEntity = file.get(root);
+
+    if (isH5WasmGroup(h5wEntity)) {
+      // Build absolute paths since .paths() are relative
+      return h5wEntity.paths().map((path) => `${root}${path}`);
```

Comment:
Are we sure that we want absolute paths? Or did we discuss this already?

Reply:
> Or did we discuss this already?

Not really. But we have to somehow get the absolute path at some point if we want to reach for the entity afterwards. So it made sense for me to use absolute paths from the get-go.

Reply:
ah yeah, of course, fair point :+1: